.gitignore

```diff
@@ -2,4 +2,7 @@ out
 node_modules
 npm-debug.log
 dist
-.vscode-test
\ No newline at end of file
+.vscode-test
+# output of simulator bundling
+resources/simulator/scripts/**/*
```

Comment:
will these resources be automatically deleted once simulation is stopped?

Reply:
So these are built at package / compile time - It's essentially running a bundle of our application code and vendor dependencies into mini website / webviews for the extension. This webview is the entire "Send D2C message" feature of the extension.

This way we can easily update and track dependencies in them instead of just having prebuilt / minified javascript embedded in the extension.